components/AdminItem/AdminItem.module.css

```diff
@@ -0,0 +1,130 @@
+.list-item {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 68px;
+  padding-left: 2%;
+  padding-right: 2%;
+  border-top: 1px solid #EBEBEB;
+  color: #646464;
+}
+
+.name {
+  width: 20%;
+  /* min-width: 150px; */
+  min-width: 200px;
+}
+
+.role {
+  width: 15%;
+  /* min-width: 150px; */
+  min-width: 100px;
+}
+
+.email {
+  width: 20%;
+  /* min-width: 150px; */
+  min-width: 200px;
+}
+
+.phone {
+  /* width: 15%; */
+  min-width: 150px;
+  /* min-width: 310px; */
+}
+
+.buttons {
+  /* width: 30%; */
+  display: flex;
+  /* justify-content: space-evenly; */
+  min-width: 310px;
+}
+
+.buttons-header {
+  width: 25%
+}
+
+.action {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 80px;
+  height: 30px;
+  background-color: #EDEDED;
+  border: 1px solid #C8C8C8;
+  border-radius: 16.5px;
+  margin-right: 10px;
+  font-size: 14px;
+  font-family: Avenir;
+  /* font-weight: 800; */
+  color: #646464;
+}
+
+.action2 {
```

Comment:
nit: please rename the CSS rules (sorry for not catching these earlier), use more descriptive rule names for these.